monkestation/code/__DEFINES/mobs.dm

```diff
@@ -0,0 +1 @@
+#define SPECIES_ARACHNIDS "arachnid"
```

Comment:
due to how preprocessor defines work, make a `~monkestation` folder in __DEFINES of the core tg folder